install-dependencies.sh

```diff
@@ -80,73 +81,140 @@ if [ -z "$DISTRO" ]; then
       DISTRO="mandriva"
    elif [ -f /etc/debian_version ]; then
       DISTRO="debian"
+   elif [ -f /etc/solus-release ]; then
+      DISTRO="solus"
    elif [ -f /usr/local/Homebrew/bin/brew ]; then
       DISTRO="homebrew"
    fi
 fi
 
+asio_not_packaged() {
+   echo
+   echo "Asio is not packaged for $1, please install it from source:"
+   echo " - Download the latest stable version from"
+   echo "      https://sourceforge.net/projects/asio/files/asio/"
+   echo " - Unpack in your development directory"
+   echo " - Run './configure --prefix=/usr && $2 make install-data' in the unpacked asio"
+   echo "   directory (or just copy recursively 'asio.hpp' and the 'asio' subdirectory"
+   echo "   from 'include' to '/usr/include')"
```

Comment:
Should be changed to recommend `auto_dependencies`, but I think documenting system-wide install should also stay.